atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -171,6 +172,25 @@ def set_go_metadata(self, extra):
             extra['image']['go'] = go
 
     def set_operators_metadata(self, extra, worker_metadatas):
+        # upload metadata from bundle (part of image)
+        op_bundle_metadata = self.workflow.prebuild_results.get(PLUGIN_PIN_OPERATOR_DIGESTS_KEY)
+        if op_bundle_metadata:
```

Comment:
no need to recreate the whole thing

since _get_replacement_pullspecs will already have converted ImageName to str, just use exactly what plugin pre_pin_operator_digests returned

Reply:
I used this as gating to prevent unexpected metadata do koji. pre pin plugin already returns more and may be extended.